src/vue-froala.js

```diff
@@ -89,20 +91,20 @@ export default (Vue, Options = {}) => {
         if (this.editorInitialized) {
           return;
         }
-
         this.currentConfig = this.config || this.defaultConfig;
-
-        this._$element = jQuery(this.$el);
-        
-        
         
+        this._$element = this.$el;
```

Comment:
As we're not using jQuery anymore, I would want to change `_$element` with the instance of the editor.